vue/components/organisms/ripe-image/ripe-image.vue

```diff
@@ -706,11 +710,6 @@ export const RipeImage = {
         this.image.update(this.state);
     },
     methods: {
-        setInitials(initials, profiles) {
-            const _initials = initials || "$empty";
-            const _profiles = profiles || null;
-            this.ripeData.setInitials(_initials, _profiles);
-        },

```

Comment:
@gcandal The removal of this method breaks the component behaviour.
Also, I have a question. `profiles` and `engraving` should do the same thing which is update the initials by calling `setInitials` right? But as they are two different contexts they should be both kept as props right? profiles as an `array` and engraving as a `string`

Reply:
Conclusion: use some kind of initialsBuilder in WYSYWIG.

```
_defaultInitialsBuilder(initials, engraving, element) {
            return {
                initials: initials || "$empty",
                profile: PROFILES_INPUT_VALUE
            };
        },
```